SumoLogic.Logging.Log4Net/BufferedSumoLogicAppender.cs

```diff
@@ -268,6 +269,12 @@ protected override void Append(LoggingEvent loggingEvent)
                 return;
             }
 
+            if(Layout == null)
+            {
+                this.LogLog.Warn("No Layout was defined. Falling back to default.");

```

Comment:
Personally, Id prefer this be a bit more explicit since this will be inside of a dll, and it will be hard to discover where this is coming from. 

`No Log4net layout defined for SumoLogic.Logging.Log4Net. Failing back to a default layout.`